### PR TITLE
Added outreachy internship

### LINKED ISSUE #396: adding outreachy internship under the internships section
**Updating the internship section.**

Screenshot (129)

**Adding outreachy internship as one of the internships under this section**

### PR DESCRIPTION
**Pull request by mankaache**
**I added outreachy internship to the internship section of the website**
1. I introduced outreachy and briefly outlined what it is about and included a link to the outreachy website
2. I mentioned the different Talawa projects that are part of the outreachy internship with links to the github repositories
3. I added a getting-started page where i outlined the different files applicants should read through to understand how to contribute
4. Some links to different parts of the website were included and also a link to the slack channel

**Issue Number:**

Fixes #396

### CODE REVIEW

File: docs/internships/outreachy/getting-started.md [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+5. **ISSUE_GUIDELINES.md**
+
+The steps to create a new issue and how you can work with issues are outlines in this file. For instance, applicants should only work on issues assigned to them.
+
+6. **PR_GUIDELINES.md**
+
+Pull request guidelines are stated here and should be followed to the letter. The steps to open an acceptable pull request are also outlined here.
+
+7. **README.md**
```
Comment (palisadoes) [inline docs/internships/outreachy/getting-started.md]: 1. Please put this as the first point
2. Mention as a bullet that people can join our slack channel by following the link on the README.md file of the Talawa repo

File: docs/internships/outreachy/getting-started.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+As part of the application process, all applicants must make at least one contribution to be accepted as an intern for this project. Applicants can contribute to our project through our github repositories.
+1. [Talawa mobile github](https://github.com/PalisadoesFoundation/talawa)
+1. [Talawa API github](https://github.com/PalisadoesFoundation/talawa-api)
+1. [Talawa Admin github](https://github.com/PalisadoesFoundation/talawa-admin)
+1. [Talawa Docs github](https://github.com/PalisadoesFoundation/talawa-docs).
+## Getting started with our repositories
+
+In our github repositories we have a few files that include guidelines and rules to guide applicants on how to contribute and the rules to follow in other to open a acceptable pull request and give a meaningful contribution.
+
+1. **CODE_OF_CONDUCT.md**
```
Comment (palisadoes) [inline docs/internships/outreachy/getting-started.md]: 1. Create an H2 title `Important Files` then...
1. Make each of these bullets H3 titles under that

File: docs/internships/outreachy/getting-started.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+title: Our repositories
+sidebar_label: Getting Started
+---
+
+As part of the application process, all applicants must make at least one contribution to be accepted as an intern for this project. Applicants can contribute to our project through our github repositories.
+1. [Talawa mobile github](https://github.com/PalisadoesFoundation/talawa)
+1. [Talawa API github](https://github.com/PalisadoesFoundation/talawa-api)
+1. [Talawa Admin github](https://github.com/PalisadoesFoundation/talawa-admin)
+1. [Talawa Docs github](https://github.com/PalisadoesFoundation/talawa-docs).
+## Getting started with our repositories
```
Comment (palisadoes) [inline docs/internships/outreachy/getting-started.md]: 1. Capitalize each word
1. Add introductory text beneath the title.

File: docs/internships/outreachy/getting-started.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
+---
+id: getting-starteds
+title: Our repositories
+sidebar_label: Getting Started
+---
+
+As part of the application process, all applicants must make at least one contribution to be accepted as an intern for this project. Applicants can contribute to our project through our github repositories.
+1. [Talawa mobile github](https://github.com/PalisadoesFoundation/talawa)
```
Comment (palisadoes) [inline docs/internships/outreachy/getting-started.md]: Add a space above. Sometimes markdown requires that for bullets to be recognised.

File: docs/internships/outreachy/getting-started.md [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+1. [Talawa API github](https://github.com/PalisadoesFoundation/talawa-api)
+1. [Talawa Admin github](https://github.com/PalisadoesFoundation/talawa-admin)
+1. [Talawa Docs github](https://github.com/PalisadoesFoundation/talawa-docs).
```
Comment (palisadoes) [inline docs/internships/outreachy/getting-started.md]: Add a closing sentence below the bullets. It makes the ending less abrupt

File: docs/internships/outreachy/introduction.md [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+---
+
+Outreachy is a paid, remote internship program. Outreachy's goal is to support people from groups underrepresented in tech. We help newcomers to free software and open source make their first contributions.
+Outreachy provides internships to work open source. People apply from all around the world. Interns work remotely, and are not required to move. [Outreachy internship](https://www.outreachy.org/) projects may include programming, user experience or documentation.
+
+The outreachy internship application has two phases, the first stage is called [initial application](https://www.outreachy.org/docs/applicant/#initial-application) and the second stage is called the [contribution period](https://www.outreachy.org/docs/applicant/#application-prep). 
+
+### Eligibility Criteria
+
+Inorder to be part of outreachy internship,
```
Comment (palisadoes) [inline docs/internships/outreachy/introduction.md:13]: Please add  a link to the criteria. That way we won't have to maintain it if they change. Reduce the amount of text to match. Add the link to the section below.

File: docs/internships/outreachy/introduction.md [hunk 7]
```diff
@@ -0,0 +1,29 @@
...
+
+1. You must be 18 years of age or older by May 29, 2023
+1. You must be available for a full-time internship. Outreachy interns work 30 hours per week.
+1. You must not be a past Outreachy intern.
+1. You must not be a past Outreach Program for Women intern. 
+
+
+Find out more about the [eligibility](https://www.outreachy.org/docs/applicant/) criteria of outreachy internship.
+
+### some useful links
```
Comment (palisadoes) [inline docs/internships/outreachy/introduction.md]: Capitalize

File: docs/internships/outreachy/contribution.md [hunk 8]
```diff
@@ -0,0 +1,32 @@
...
+
+To contribute to the our projects,
+
+1. Fork the repository by clicking on the fork button on the repository. This creates a copy of the repository to the application github account
+1. Clone the forked (copied) repository
+1. Follow the installation guide and install required depencies
+1. Go through the project and open an issue if you notice a bug pr feature you would want to be added.
+1. Once the issue is assigned to you, 
+1. Create a new branch and make your changes to the project 
+1. Open a pull request(PR)
```
Comment (palisadoes) [inline docs/internships/outreachy/contribution.md]: Add a closing sentence after these and any other bullets in the PR. It makes the flow better.

File: docs/internships/outreachy/contribution.md [hunk 9]
```diff
@@ -0,0 +1,32 @@
...
+id: contribution-process
+title: Contribution Process
+sidebar_label: Contribution Process
+---
+
+### Introduction
+
+The Palisadoes Foundattion has a sponsored work on Talawa projects for some years now. Talawa is basically an open-source project to help community-based organizations such as clubs, neighborhood groups,religious institutions etc manage their daily activities.
+The Talawa applications has 3 components
+1. Talawa-API
```
Comment (palisadoes) [inline docs/internships/outreachy/contribution.md:13]: Add a space above.

**Review by palisadoes — CHANGES_REQUESTED** [review]
You never know what you missed until you get it. Thanks.
Please make the necessary changes.

**Comment by mankaache** [comment]
> You never know what you missed until you get it. Thanks. Please make the necessary changes.
Thank you for the review I will make the necessary changes.

File: docs/internships/outreachy/introduction.md [hunk 10]
```diff
@@ -0,0 +1,29 @@
...
+
+1. You must be 18 years of age or older by May 29, 2023
+1. You must be available for a full-time internship. Outreachy interns work 30 hours per week.
+1. You must not be a past Outreachy intern.
+
+Find out more about the [eligibility](https://www.outreachy.org/docs/applicant/) of outreachy internship.
+
+### Some Useful Links
+
+here are some links to get you started
```
Comment (palisadoes) [inline docs/internships/outreachy/introduction.md:23]: Capitalize H

**Review by palisadoes — APPROVED** [review]

**Comment by mankaache** [comment]
Thank you very much :)

**Pull request merged**